### ISSUE TITLE
cmd + fn + 3 does not open reactlog in shiny apps run in the viewer pane

### ISSUE THREAD

**Issue by kierisi** [rstudio/rstudio#9757]
### System details

    RStudio Edition : Desktop
    RStudio Version : 1.4.1098
    OS Version      : macOS 11.2.3
    R Version       : 4.0.3

### Steps to reproduce the problem

```
install.packages("shiny"); 
library(shiny); 
options(shiny.reactlog = TRUE); 
runExample("01_hello")
```
Click to populate the reactlog, then use the keyboard shortcut `CMD + fn + F3`.

### Describe the problem in detail
The keyboard shortcut `CMD + fn + F3` works when the shiny app is run in window or externally, but the keyboard shortcut does not open the shiny reactlog when the app is run in the viewer pane.

This is similar to the problem outlined in [this issue](https://github.com/rstudio/rstudio/issues/4045).

### Describe the behavior you expected
When running the app in the Viewer Pane I expect the reactlog to be opened in an additional window or pane.

- [x] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [ ] If I am reporting an RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by ronblum** [comment]
@kierisi Thank you for raising this! The current version, and specifically for the 1.4 release, is [Juliet Rose 1.4.1717](https://www.rstudio.com/products/rstudio/download/#download). Does the issue still occur with it?

If you're interested, we're currently working on Ghost Orchid. [Daily builds for Ghost Orchid are available](https://dailies.rstudio.com), but note that these are for testing purposes only and not for production use.

**Comment by kierisi** [comment]
@ronblum any time! I updated R to Juliet Rose this morning and confirmed that the issue still persists for the viewer pane mode only. this isn't a blocker for my work, and I appreciate the suggestion of Ghost Orchid!

**Comment by romainfrancois** [comment]
I *think* this is one of those issues fixed by electron. At least it worked for me with build 547 from https://dailies.rstudio.com/rstudio/spotted-wakerobin/electron/macos/ and it did not with the qt build.

**Comment by ronblum** [comment]
[Note: to reproduce, first install `reactlog` and run `options(shiny.reactlog = TRUE)` in the Console Pane.]

Verified fixed in RStudio Desktop 2022.11.0+91 and R 4.2.0/1 on
- MacOS 12.5 with <kbd>Cmd-Fn-3</kbd>
- Windows 11 with <kbd>Ctrl-Fn-3</kbd>

Using
- Viewer Pane
- Widow
- Browser
